channel-diagrams.tex

```diff
@@ -123,15 +123,15 @@
   \path (-leftp3) -- +({180-\angle}:\clearance mm) coordinate (-Leftp3);
 
   \begin{scope}[-, every to/.style={out={-\angle}, in={180-\angle}}]
-    \draw (-Leftp1) to (-leftp2);
+    \draw (-Leftp1) to (-leftp1);
     \draw (-Leftp2) to (-leftp3);
-    \draw (-Leftp3) to (-leftp1);
+    \draw (-Leftp3) to (-leftp2);
```

Comment:
The changes to this file redefine `guarantee` (used only in Figure 25 and 26) and `guarantee2other` (used only in Figure 26) diagrams. In essence the change swaps the outcome of `G` and `G'`: so that the funding from (A,I) eventually prioritises A first in J (and likewise for B). In other words (during opening) the left guarantee now has output ports 132 and the right guarantee has output ports 231. The necessary follow on change is made to Figure 26 when G has been defunded.

This is a necessary change given the reordering of A and B in J.

Reply:
I have just spotted another issue, however, in that L and L' seemed to have switched positions in Figure 26 (w.r.t Figure 25), as well as the ordering of A and B in J. I think we ought to switch L and L' back.

Reply:
7fd8d55